src/main/java/com/ericsson/ei/notifications/JenkinsCrumb.java

```diff
@@ -95,7 +95,9 @@ public JsonNode fetchJenkinsCrumb(String encoding, String url)
      */
     private URL buildJenkinsCrumbUrl(String url) throws MalformedURLException {
         String baseUrl = urlParser.extractBaseUrl(url);
-        URL jenkinsCrumbUrl = new URL(baseUrl + JENKINS_CRUMB_ENDPOINT);
+        String contextPath = urlParser.extractContextPath(url).split("/")[1];
```

Comment:
This will break functionality when its deployed with no web-server context-paths, then you will get first part of Jenkins entrypoint paths as context-path, which we do in several deployments, e.g. in Docker and Kubernetes and Integration-Tests. I think also in frontend Integrations-tests.
So you need to make a smarter solution that support with and without web-server context-paths.